atom/browser/api/lib/browser-window.coffee

```diff
@@ -40,8 +40,9 @@ BrowserWindow::_init = ->
   @once 'closed', =>
     BrowserWindow.windows.remove @id if BrowserWindow.windows.has @id
 
-BrowserWindow::openDevTools = ->
-  @_openDevTools()
+BrowserWindow::openDevTools = (options={}) ->
+  dock = if options.detach then false else true
+  @_openDevTools(dock)
```

Comment:
since its optional, wouldnt it be better to not pass `undefined` ? otherwise i have to put up check for it on c++ land right.